cross/micronucleus/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+
+github.setup        micronucleus micronucleus 2.5 v
+
+categories          cross
+platforms           darwin
+maintainers         {@mholling gmail.com:mdholling} openmaintainer
+description         Micronucleus is a bootloader designed for AVR ATtiny microcontrollers
+long_description    Micronucleus is a bootloader designed for AVR ATtiny microcontrollers \
+                    with a minimal usb interface, cross platform libusb-based program \
+                    upload tool, and a strong emphasis on bootloader compactness.
+homepage            https://github.com/micronucleus/micronucleus
```

Comment:
Since you are using the `github` portgroup, you don't need to explicitly specify the homepage.  It will be automatically generated off of `github.setup`.